tokio-sync/src/oneshot.rs

```diff
@@ -498,6 +499,7 @@ impl State {
         State(0)
     }
 
+    #[allow(clippy::trivially_copy_pass_by_ref)]
```

Comment:
Thanks. I'm pretty sure all of these can / should be fixed. We can handle them later. I added a note in #1083.